src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -1018,8 +1020,6 @@ INSERT INTO tab_processes VALUES('ActiveTransportIntrinsic_MM','PROCESS',1,'ACTI
 INSERT INTO tab_processes VALUES('ActiveTransportSpecific_Hill','PROCESS',1,'ACTIVE_TRANSPORT','Hill','ActiveTransport','TRANSPORT',0);
 INSERT INTO tab_processes VALUES('ActiveTransportSpecific_MM','PROCESS',1,'ACTIVE_TRANSPORT','MM','ActiveTransport','TRANSPORT',0);
 INSERT INTO tab_processes VALUES('BiliaryClearance','PROCESS',1,'SYSTEMIC_PROCESSES','FirstOrder','Secretion','TRANSPORT',0);
-INSERT INTO tab_processes VALUES('BiliaryClearanceToDuodenum','PROCESS',1,'SIMULATION_ACTIVE_PROCESS','FirstOrder','Secretion','TRANSPORT',0);
-INSERT INTO tab_processes VALUES('BiliaryClearanceToGallBladder','PROCESS',1,'SIMULATION_ACTIVE_PROCESS','FirstOrder','Secretion','TRANSPORT',0);

```

Comment:
@Yuri05 Basically replace one by the other:
BiliaryClearanceToDuodenum=>LiverActiveEffluxToDuodenum_FirstOrder
BiliaryClearanceToGallBladder =>LiverActiveEffluxToGallbladder_FirstOrder